### PR TITLE
Improvement / HOLO 1369 Holograph Registry Write Functions

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

Changes made to the package: **sdk**.

I made it better by doing ...

- add write functions to `Registry` contract class.
- add missing read functions to `Registry` contract class.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All tests are passing

### CODE REVIEW

File: packages/sdk/src/contracts/registry.contract.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import {getContract} from 'viem'
+import {AbiParameterToPrimitiveType, Hex, getContract} from 'viem'
```
Comment (feliperodriguess) [inline packages/sdk/src/contracts/registry.contract.ts:1]: You didn't use the following import: `AbiParameterToPrimitiveType`

File: packages/sdk/src/contracts/registry.contract.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 import {Network} from '@holographxyz/networks'
-import {Address, ExtractAbiFunctionNames} from 'abitype'
+import {AbiParametersToPrimitiveTypes, Address, ExtractAbiFunctionNames} from 'abitype'
```
Comment (feliperodriguess) [inline packages/sdk/src/contracts/registry.contract.ts:3]: Neither here

File: packages/sdk/src/utils/contracts.ts [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import {Network, getNetworkByChainId} from '@holographxyz/networks'
-import {Abi, AbiFunction} from 'viem'
+import {Abi, AbiFunction, AbiParameterToPrimitiveType} from 'viem'
+
+// type primitiveType = AbiParameterToPrimitiveType<{name: 'test', type: 'uint32'}> // NOTICE: use this to figure out which primitive type to use
```
Comment (feliperodriguess) [inline packages/sdk/src/utils/contracts.ts:4]: This is so useful 🚀

**Review by feliperodriguess — APPROVED** [review]

**Review by alexanderattar — APPROVED** [review]

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**